interfaces/HO-Portal/src/app/program/export-list/export-list.component.ts

```diff
@@ -84,7 +84,9 @@ export class ExportListComponent implements OnChanges {
           this.exportType === ExportType.selectedForValidation) ||
         (activePhase.name === ProgramPhase.payment &&
           this.exportType === ExportType.payment &&
-          this.paymentExportAvailable)) &&
+          this.paymentExportAvailable)) ||
+      (activePhase.name === ProgramPhase.payment &&
+        this.exportType === ExportType.unusedVouchers) &&
       this.userRole === UserRole.ProgramManager
     );
   }
```

Comment:
Hmm.. I think all this logic shouldn't be handled inside this component..

But for now, this seems like the next-best-place to combine everything...